app/src/main/java/com/eveningoutpost/dexdrip/watch/miband/MiBandService.java

```diff
@@ -164,20 +151,120 @@ public QueueMessage(String functionName, String message_type, String message, St
         startBgTimer();
     }
 
+    private static final boolean isBetweenValidTime(Date startTime, Date endTime, Date currentTime) {
+        //Start Time
+        Calendar StartTime = Calendar.getInstance();
+        StartTime.setTime(startTime);
+        StartTime.set(1, 1, 1);
+
+        Calendar EndTime = Calendar.getInstance();
+        EndTime.setTime(endTime);
+        EndTime.set(1, 1, 1);
+
+        //Current Time
+        Calendar CurrentTime = Calendar.getInstance();
+        CurrentTime.setTime(currentTime);
+        CurrentTime.set(1, 1, 1);
+        if (EndTime.compareTo(StartTime) > 0) {
+            if ((CurrentTime.compareTo(StartTime) >= 0) && (CurrentTime.compareTo(EndTime) <= 0)) {
+                return true;
+            } else {
+                return false;
+            }
+        } else if (EndTime.compareTo(StartTime) < 0) {
+            if ((CurrentTime.compareTo(EndTime) >= 0) && (CurrentTime.compareTo(StartTime) <= 0)) {
+                return false;
+            } else {
+                return true;
+            }
+        } else {
+            return false;
+        }
+    }
+
+    // Mega Status
+    public static List<StatusItem> megaStatus() {
+
+        final List<StatusItem> l = new ArrayList<>();
+        final Inst II = Inst.get(MiBandService.class.getSimpleName());
```

Comment:
Please use understandable and traceable variable names.

Reply:
Thanks, but this is just a copypaste from another places of xdrip, like src\main\java\com\eveningoutpost\dexdrip\watch\thinjam\BlueJayService.java , function megaStatus. I used the same variable names pattern to maintain the overall application code style.